src/helm/benchmark/run_expander.py

```diff
@@ -819,6 +819,57 @@ def expand(self, run_spec: RunSpec) -> List[RunSpec]:
             )
             for value in self.values
         ]
+    
+    
+class ChatMLRunExpander(RunExpander):
+    
+    name = "chatml"
+    
+    def __init__(self):
+        self.name = type(self).name
+
+    def expand(self, run_spec: RunSpec) -> List[RunSpec]:
+        adapter_spec = run_spec.adapter_spec
+        # according to https://github.com/openai/openai-python/blob/main/chatml.md#few-shot-prompting
+        # few-shot examples should do `<|im_start|>system name=example_user` or `<|im_start|>system name=example_assistant`
+        # but it is also possible to put examples into a user message.
+        if run_spec.name.split(':')[0] in ('msmarco',):
```

Comment:
Could we set `scenario_name = run_spec.name.split(':')[0]`?  Ideally, we wouldn't have to hard-code the scenario names here, but I don't know of any short-term solutions.  I think longer-term, we'll want to make the inputs more structured, which would help.